coa-helm/templates/postgres.yaml

```diff
@@ -0,0 +1,35 @@
+{{ include "terra-batch-libchart.postgres-deploy" (list . "coa-helm.postgres-deploy") }}
+{{ define "coa-helm.postgres-deploy" }}
+# Overrides go here
+{{ end }}
+
+---
+
+{{ include "terra-batch-libchart.postgres-service" (list . "coa-helm.postgres-service") }}
+{{ define "coa-helm.postgres-service" }}
+# Overrides go here
+{{ end }}
+
+---
+
+{{ include "terra-batch-libchart.postgres-config" (list . "coa-helm.postgres-config") }}
+{{ define "coa-helm.postgres-config" }}
+# Overrides go here
+{{ end }}
+
+---
+
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
I don't think this is a great place for this, to be honest, but it felt right to keep Postgres stuff together for now.

There's a lot more that could be done to reorganize these charts